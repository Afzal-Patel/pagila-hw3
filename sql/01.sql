/*
 * Select the title of all 'G' rated movies that have the 'Trailers' special feature.
 */
