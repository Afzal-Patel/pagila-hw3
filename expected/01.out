          title          
-------------------------
 YOUNG LANGUAGE
 WEST LION
 WEREWOLF LOLA
 WARDROBE PHANTOM
 WAKE JAWS
 TYCOON GATHERING
 TRUMAN CRAZY
 TRAFFIC HOBBIT
 TRACY CIDER
 TORQUE BOUND
 TEEN APOLLO
 SWEETHEARTS SUSPECTS
 STONE FIRE
 SPOILERS HELLFIGHTERS
 SPIKING ELEMENT
 SNOWMAN ROLLERCOASTER
 SLEEPLESS MONSOON
 SHINING ROSES
 SECRETS PARADISE
 SAGEBRUSH CLUELESS
 ROCK INSTINCT
 RINGS HEARTBREAKERS
 REBEL AIRPORT
 PULP BEVERLY
 PRIMARY GLASS
 PICKUP DRIVING
 PANKY SUBMARINE
 PACIFIC AMISTAD
 OPERATION OPERATION
 NOVOCAINE FLIGHT
 NEWSIES STORY
 MOSQUITO ARMAGEDDON
 MOONWALKER FOOL
 MONTEREY LABYRINTH
 MOB DUFFEL
 MINORITY KISS
 MIDSUMMER GROUNDHOG
 MIDNIGHT WESTWARD
 MENAGERIE RUSHMORE
 MEET CHOCOLATE
 LUST LOCK
 LOVER TRUMAN
 LONELY ELEPHANT
 KILL BROTHERHOOD
 ITALIAN AFRICAN
 HYDE DOCTOR
 HOOSIERS BIRDCAGE
 HOOK CHARIOTS
 HOCUS FRIDA
 HILLS NEIGHBORS
 HEARTBREAKERS BRIGHT
 HARPER DYING
 HANGING DEEP
 GREASE YOUTH
 GRADUATE LORD
 GOLDFINGER SENSIBILITY
 GLASS DYING
 GILBERT PELICAN
 GHOST GROUNDHOG
 GARDEN ISLAND
 FLATLINERS KILLER
 FIDELITY DEVIL
 EXTRAORDINARY CONQUERER
 EVE RESURRECTION
 ENCINO ELF
 EASY GLADIATOR
 DUDE BLINDNESS
 DOWNHILL ENOUGH
 DOCTOR GRAIL
 DIVORCE SHINING
 DESTINY SATURDAY
 DAY UNFAITHFUL
 DAISY MENAGERIE
 CROSSROADS CASUALTIES
 COLDBLOODED DARLING
 CHISUM BEHAVIOR
 CHEAPER CLYDE
 CATCH AMISTAD
 CASUALTIES ENCINO
 CASABLANCA SUPER
 CAPER MOTIONS
 BULWORTH COMMANDMENTS
 BRIDE INTRIGUE
 BREAKFAST GOLDFINGER
 BLUES INSTINCT
 BLOOD ARGONAUTS
 BLANKET BEVERLY
 BIRDS PERDITION
 BEAUTY GREASE
 BAREFOOT MANCHURIAN
 AUTUMN CROW
 ARMAGEDDON LOST
 ANGELS LIFE
 ACE GOLDFINGER
(94 rows)

